bundles/org.openhab.persistence.dynamodb/src/main/java/org/openhab/persistence/dynamodb/internal/DynamoDBConfig.java

```diff
@@ -93,8 +93,10 @@ public class DynamoDBConfig {
                 String profile = (String) config.get("profile");
                 if (profilesConfigFile == null || profilesConfigFile.isBlank() || profile == null
                         || profile.isBlank()) {
-                    LOGGER.error("Specify either 1) accessKey and secretKey; or 2) profilesConfigFile and "
-                            + "profile for providing AWS credentials");
+                    LOGGER.error("""
+                            Specify either 1) accessKey and secretKey; or 2) profilesConfigFile and \
+                            profile for providing AWS credentials\
```

Comment:
Is there some extra space in here that would end up in being printed as well?

Reply:
Just checked, it is fine. The outputs of both commands look similar (except the number I added at the end)
```
[ERROR] [...] - Specify either 1) accessKey and secretKey; or 2) profilesConfigFile and profile for providing AWS credentials1
[ERROR] [...] - Specify either 1) accessKey and secretKey; or 2) profilesConfigFile and profile for providing AWS credentials2
````